develop-docs/sdk/processes/releases.mdx

```diff
@@ -132,11 +138,16 @@ Here's [an example PR] and the [follow-up to fix `fetch-depth`].
 Give the following teams access to your repository:
 
 - `engineering` -> `write`
-- `release-bot` -> `elevated bot`
 
 You can do this self-service via the settings page of your repository:
 `https://github.com/getsentry/REPONAME_HERE/settings/access`
 
+## Create ruleset for the repo
+
+Download and save the [default ruleset template](https://docs.sentry.io/json/Default_ruleset.json) as a JSON file.
```

Comment:
This url s 404ing for me, is it correct?

Reply:
Ah, I see the issue here, let's change it to a relative url:

```suggestion
Download and save the [default ruleset template](/json/Default_ruleset.json) as a JSON file.
```

Reply:
@coolguyzone the changes I'm making to are on develop.sentry.dev, and the JSON is hosted on docs.sentry.io, hence I don't think the relative URL will work.
I would love to move the JSON over under the develop domain, but I'm not sure how to do that, any suggestions?

Reply:
Ah, fair point. In that case, the absolute URL makes sense.

Reply:
I was having incorrect assumption here, I thought the JSON folder is only available under docs.sentry.io, but it's also available for develop.sentry.dev. 
Updated link to make it a relative URL